README.md

```diff
@@ -149,3 +150,81 @@ PostgreSQL configuration file after the benchmark completes.
 Some options can be passed on the command line and in a config file.
 Any such option passed on the command line will override the value set in the
 config file.
+
+
+# HTAP benchmark
+
+A mixed workload benchmark using a hybrid TPC-C/TPC-H schema is available in `benchmarks/htap`.
+Data preparation is identical to the other benchmarks (see "Creating a database and loading data"
+above).
+
+The HTAP benchmark requires command line arguments that differ from the ones described above.
+The `--dsn` argument is shared with the other benchmarks and must be provided.
+The `--benchmark` argument is not used, instead the name `htap` must be provided directly after the `--dsn` argument.
```

Comment:
I pushed a new commit that improves this. Howover, it now requires that the benchmark name (prefixing it with `--benchmark` is now deprecated but can still be used) come directly after `--dns`. This is how we always do it, but it might break existing code that did it in a different order.